CustomFileFormats/DynamicInputTextures.md

```diff
@@ -0,0 +1,136 @@
+# Dolphin Dynamic Input Textures Specification (v1)
```

Comment:
Just like `ResourcePacks.md`, this should probably go in the `docs/` folder.

Reply:
Will do, thank you!